### ISSUE TITLE
Document/handle slot sharing for locals

### ISSUE THREAD

**Issue by provegard** [provegard/ncdbg#83]
Hovering over a local variable may show the contents of another local variable. I think this is due to that the JVM optimizes slot usage. Check whether or not this is the case and document it (or work around it if possible).

**Comment by benoitcerrina** [comment]
I saw this happening too.  But not always.

**Comment by benoitcerrina** [comment]
note that when this happens it is not just the hover which is incorrect.  Typing the local variable name in the console will return the wrong value.  
The "Local" tree was still showing the correct values in front of the correct name though

**Comment by provegard** [comment]
Yes I saw the same. I don't think there is much to do. When I added forced uses of all locals, this behavior disappeared.
